scripts/utils.py

```diff
@@ -12,7 +12,7 @@
 from electricitymap.contrib.lib.types import ZoneKey
 
 ROOT_PATH = pathlib.Path(__file__).parent.parent
-LOCALES_FOLDER_PATH = ROOT_PATH / "web/public/locales/"
+LOCALES_FOLDER_PATH = ROOT_PATH / "web/src/locales/"
```

Comment:
This one should stay public, it's used in the remove_zone.py script

Reply:
Shouldn't it be both then?
We would have to loop over them but it shouldn't be much of an issue.

Reply:
Ideally yes, but I would imagine it's less of a problem to have old keys in the app-only-translations - and since this is only somewhat temporary, I think it's okay to not do both :)